buildcheck/views/navbar.py

```diff
@@ -5,8 +5,8 @@ def navbar():
     return rx.flex(
         rx.badge(
             rx.icon(tag="table-2", size=28),
-            rx.heading("Customer Data App", size="6"),
-            color_scheme="green",
+            rx.heading("ARCH", size="6"),
+            color_scheme="crimson",
             radius="large",
```

Comment:
This file has moved to `buildcheck/components/navbar.py`, please use that instead after merging master in.